CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+1. **Please sign one of the contributor license agreements below!**
+1. Fork the repo, develop and test your code changes, add docs.
+1. Make sure that your commit messages clearly describe the changes.
+1. Send a pull request.
+
+## Opening an issue
+
+If you've tried everything in our [Cloud SQL Proxy Client
```

Comment:
I think we should reword this - we want it to be clear that this forum is for discussing/troubleshooting the proxy itself, and that any issues with Cloud SQL should open a support ticket.